packages/ui/src/features/data-points/useDatapoints.ts

```diff
@@ -140,7 +135,10 @@ const _addRow = (original: DataEntry[], newRow: DataEntry) =>
       data: newRow.data.filter(dp => dp.value !== undefined),
       meta: {
         ...newRow.meta,
-        id: original.reduce((id, curr) => Math.max(id + 1, curr.meta.id), 1),
+        id:
```

Comment:
Fixes a bug where data points would get the same id if data points had been deleted from the table.